robottelo/ui/base.py

```diff
@@ -429,6 +429,9 @@ def auto_complete_search(self, go_to_page, entity_locator, partial_name,
         self.wait_for_ajax()
         strategy, value = common_locators['auto_search']
         self.click((strategy, value % name))
+        # Intentionally clicking second time because sometimes first time
+        # click just selects/highlights the item but not clicking it
+        self.click((strategy, value % name))
```

Comment:
Also another reason, why I feel this shouldn't be enabled by default, https://github.com/SeleniumHQ/selenium/issues/1202#issuecomment-152121503